src/GraphQL/Utilities/SchemaDirectiveVisitor.cs

```diff
@@ -1,38 +0,0 @@
-using System;
-using System.Collections.Generic;
-
-namespace GraphQL.Utilities
-{
-    public abstract class SchemaDirectiveVisitor : BaseSchemaNodeVisitor
-    {
-        public abstract string Name { get; }
-
-        public Dictionary<string, object> Arguments { get; set; } = new Dictionary<string, object>();
```

Comment:
Important! The visitor was stateful because of this. This is no longer the case.